src/styles/blocks/card.scss

```diff
@@ -0,0 +1,95 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  background-color: $white-color;
+  border: 1px solid $gray-elements-color;
+  border-radius: 5px;
+  padding: 32px 0 16px;
+  padding-inline: 16px;
+  box-sizing: border-box;
+  width: 200px;
+
+  &__image {
+    display: flex;
+    align-items: center;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+    background-position: center;
+  }
+
+  &__description {
+    display: flex;
+    flex-direction: column;
+    gap: 4px;
+    padding-bottom: 16px;
```

Comment:
1. use margin to make outer indent between elements 
2. remove gap and use margin both for title and code